test/ui-testing/pick-lists.js

```diff
@@ -0,0 +1,268 @@
+            }
+          }, editedPickList)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should not find pick list ${pickList}`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .wait('#editList-pick-lists')
+          .evaluate(list => {
+            const row = document.evaluate(
+              `//*[@id="editList-pick-lists"]//div[.="${list}"]`,
+              document,
+              null,
+              XPathResult.FIRST_ORDERED_NODE_TYPE
+            ).singleNodeValue;
+            if (row != null) {
+              throw Error(`Should not find row with list ${list}`);
+            }
+          }, pickList)
+          .then(done)
+          .catch(done);
+      });
+
+
+      it(`should create pick list value ${pickListValue} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-list-values"]')
+          .click('a[href="/settings/licenses/pick-list-values"]')
+          .waitUntilNetworkIdle(1000)
+          .wait('#categorySelect')
+          .type('#categorySelect', editedPickList)
+          .wait('#clickable-add-pick-list-values')
+          .click('#clickable-add-pick-list-values')
+          .wait('input[name="items[0].label"]')
+          .type('input[name="items[0].label"]', pickListValue)
+          .wait('#clickable-save-pick-list-values-0')
+          .click('#clickable-save-pick-list-values-0')
+          .wait(222)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should find pick list value ${pickListValue} in list`, done => {
+        nightmare
+          .wait('#editList-pick-list-values')
+          .evaluate(value => {
+            const valueRows = [...document.querySelectorAll('#editList-pick-list-values')].map(e => e.textContent);
+            const valueElement = valueRows.find(r => r.indexOf(value) >= 0);
+            if (!valueElement) {
+              throw Error(`Could not find row with the edited value ${value} but found ${valueElement}`);
+            }
+          }, pickListValue)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should edit pick list value ${pickListValue} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-list-values"]')
+          .click('a[href="/settings/licenses/pick-list-values"]')
+          .waitUntilNetworkIdle(1000)
+          .wait('#categorySelect')
+          .type('#categorySelect', editedPickList)
+          .wait('#clickable-edit-pick-list-values-0')
+          .click('#clickable-edit-pick-list-values-0')
```

Comment:
I don't have the same concern here as [the pick list test](https://github.com/folio-org/ui-licenses/pull/200/files#diff-b28fa7da29a06fb99b5cfa0f5ded3788R74) since we're only picking values for the pick list we just created, so this is fine.